app/views/hyrax/base/_form_relationships.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if Flipflop.assign_admin_set? %>
   <%= f.input :admin_set_id, as: :select,
-      collection: Hyrax::AdminSetService.new(controller).select_options(:deposit),
-      include_blank: true,
+      include_blank: false,
+      collection: Hyrax::AdminSetOptionsPresenter.new(Hyrax::AdminSetService.new(controller)).select_options,
```

Comment:
I don't disagree; this PR merges the latest stuff from Sufia over and does not otherwise introduce new code or patterns. Inclined to take this up later. Assuming it passes. ;)